.reuse/dep5

```diff
@@ -3,50 +3,18 @@ Upstream-Name: xNVMe
 Upstream-Contact: Simon A. F. Lund <os@safl.dk>
 Source: https://github.com/OpenMPDK/xNVMe
 
-Files: .github/codeql/codeql-config.yml .github/workflows/verify.yml .pre-commit-config.yaml meson_options.txt xnvme.spec.in
+Files: *
```

Comment:
Is this to say that all files have Samsung copyright and BSD-3-Clause license?
In that case you could probably remove the license from the lines below.

Reply:
I see. What I was proposing is removing the license lines from the code below, since it is the same license as the *. But perhaps `lintian` doesn't understand that.
```
Files: lib/xnvme_be_fbsd_async.c
Copyright: 2023 Fedor Uporov <fuporov.vstack@gmail.com>
License: BSD-3-Clause

Files: toolbox/third-party/windows/sys/queue.h
Copyright: 1991, 1993, The Regents of the University of California.
License: BSD-3-Clause

Files: toolbox/xnvme-driver.sh
Copyright: 2016 Intel Corporation
License: BSD-3-Clause
```

Reply:
Right ok, yeah, I don't think that will work.